tests/playwright/sanity/container.test.js

```diff
@@ -347,11 +352,18 @@ test.describe( 'Container tests', () => {
 		const videoURL = await page.locator( '.attachment-details-copy-link' ).inputValue(),
 			editor = await wpAdmin.useElementorCleanPost(),
 			containerId = await editor.addElement( { elType: 'container' }, 'document' ),
-			container = editor.getFrame().locator( '.elementor-element-' + containerId );
+			container = editor.getPreviewFrame().locator( '.elementor-element-' + containerId );
 
+		// Set Canvas template.
+		await editor.useCanvasTemplate();
```

Comment:
Forcing the screenshot always to be based on the same post layout template.

Reply:
@hein-obox Why Canvas and not always just the Default template? And is there other tests on the CI that changed the template and this caused other tests to fail?

Reply:
@louiswol94 

Changed.